### ISSUE TITLE
Ret changes to some gameplay pain points [#SomeChanges Suggestion]

### ISSUE THREAD

**Issue by Vaelrir** [ClassicWoWCommunity/cata-classic-bugs#1226]
TLDR:

Ret rotation feels clunky to play, current solutions to fix it during CDs at least have their own problems, a glyph to make 3 second CD Crusader Strikes obtainable fixes a lot of the rotation problems by filling in gaps.

Prepull has some major problems that affects CD usage in a major way, adding a way to get 3 Holy Power before a fight easily fixes it.

Problem 1: Rotation

Currently the general feeling of a lot of players is ret just feels like junk to play without a 3 sec CD CS or a change to the rotation to give more button presses. To viably reach this during CD times outside lust, ret currently would use a Power Infusion in normal gear.  To reach it full time for ret is pretty much impossible without a change which adds at minimum 14% increased haste (%increased from buffs/talents that's multiplicative), not from gear (additive haste rating), baseline or just to Sanctity of Battle.

The power infusion has its own problems for gameplay, giving a ret a power infusion is not going to be realistic for many players, one its extremely intuitive to give a melee class a spell haste buff, two we might not be the best target. Even so to parse its still around a 1K dps increase phase 1, Swapping ret to melee haste doesn't really fix the problem, unholy frenzy CD doesn’t line up with ret CDs, creating more problems, and melee haste aura doesn’t reach the 3s CD breakpoint passively so it does not really do anything.

Allowing ret to reach the 3 second CD breakpoint passively would be a decent change to fix some of the clunky gameplay. This can easily be done by changing the talent [Sanctity of Battle ](https://www.wowhead.com/cata/spell=25956/sanctity-of-battle)from 100% of haste effects CD to 114% of haste effects CD through a Prime glyph. The glyph could read something like “Sanctity of battle treats haste as 14% more when reducing Crusader Strike and Divine Storm cooldown.” The loss of Glyph of Exorcism and gear set differences should largely be a decent trade off to reach this, further if this is too much of a buff/nerf the glyph can be changed to compensate for the loss/gain. 14% effective bonus (multiplicative) haste seems to be the right spot where you need to make gear changes, but not too much where its just a damage loss in every situation, If this is balanced right it lets people who feel like ret is clunky be able to play it with a new but mostly same rotation as an option, while preserving the original cata ret gameplay for the people who like it. Maybe this is not enough wiggle room for the average player, 15% for swapping gear and player’s ping could potentially be better. My current estimate (this could be way off) is this would be AT MOST around a 10% increase, but this does not take into account that CS does not proc divine purpose, less crit/mastery in most sets, loss of strength. or that lust no longer really increases your damage a lot. This means it probably is lower than an increase of 10% damage. 

Gear sets are linked below for examples that reach the cap with 14% multiplicative haste and same tier sets that go for the normal mastery>crit>haste. There are some changes with gearing, reforging, and gems to reach the cap. It does not seem possible in pre raid gear. THESE ARE NOT THE BEST, THEY ARE JUST EXAMPLES.

[Phase 1 Normal Gearing] https://sixtyupgrades.com/cata/set/5Z9DjevqhLcuNaUtKTkumi
[Phase 1 Haste Gearing] https://sixtyupgrades.com/cata/set/wSYQvfu8NSLdWh5vE1aLdD
[Phase 2 Normal Gearing] https://sixtyupgrades.com/cata/set/iftFrFYUmHjycwmDA6ncJW
[Phase 2 Haste Gearing] https://sixtyupgrades.com/cata/set/94kV5g9i4h5Dh1tNwQRtW5
[Phase 3 Normal Gearing] https://sixtyupgrades.com/cata/set/3kbCuagTqmxb9NmZKHttzL
[Phase 3 Haste Gearing] https://sixtyupgrades.com/cata/set/eS7NkEWcJoD5y2UN4ohrwx

Problem 2: pre pull/on pull fix

Currently ret is either reliant on a divine purpose proc, or starting a fight with 3 Holy power from either a mob/critter to start off with a zealotry+inq to not feel like you are just wasting your pre pot if you don't get lucky or go out of your way for a parse. A very simple solution to this is just make Divine Plea give 3 HP out of combat. Could be a major glyph due to them being more resource oriented, and prot who already has this from talents could keep their more utility oriented glyphs in the major slot.

Final note:
To be clear I am not asking for buffs, just trying to propose a fix for bad feeling dead zones in ret rotation where you are on your hands and knees praying for a proc to MAYBE fill that gap.

**Comment by Vaelrir** [comment]
Something that was brought up to me, while the second one would fix the "just get lucky with a dp proc or start a fight after stacking 3 holy power" it would be op in pvp, so would need to be disabled in arenas/bgs.

**Comment by JamminL** [comment]
Archiving old non-relevant issues in preparation for MOP. If I made a mistake please comment on it.
